code/modules/mob/living/carbon/human/death.dm

```diff
@@ -43,8 +43,9 @@
 		return
 	GLOB.alive_human_list -= src
 	if(!gibbed)
-		if(HAS_TRAIT(src, TRAIT_HARDCORE))
-			status_flags |= PERMANENTLY_DEAD
+		if(HAS_TRAIT(src, TRAIT_HARDCORE) || MODE_HAS_TOGGLEABLE_FLAG(MODE_HARDCORE_PERMA))
+			if(!(species.flags & IS_SYNTHETIC)) // Synths wont perma
```

Comment:
This will make it so synths cannot perma in TRAIT_HARDCORE as well. Is that intended or an oversight?